rcpttTests/coverage/swt/button/Button Radio Test.test

```diff
@@ -6,8 +6,8 @@ Element-Type: testcase
 Element-Version: 3.0
 External-Reference: 
 Id: _UPfZgJa6EeK184kZHmPTQQ
-Runtime-Version: 1.5.6.qualifier
-Save-Time: 4/1/15 12:04 PM
+Runtime-Version: 2.5.1.202004201035
```

Comment:
Where is the test for text change when a button is deselected?

Reply:
added